src/_img/high-performance-cpp-gc/stop-the-world-sweeping.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please run these SVGs through [SVGOMG](https://jakearchibald.github.io/svgomg/) to compress them.

Reply:
Thanks, done.

Reply:
@mathiasbynens FWIW we talked with @mlippautz and SVGs need to be fixed first anyway (right now they produce a lot of space on mobile).

Reply:
I checked locally and spacing seems fine on mobile. What am I missing?

<img width="477" alt="Screen Shot 2020-05-25 at 10 02 47" src="https://user-images.githubusercontent.com/890329/82866953-31baf400-9f2a-11ea-95b5-0faa7c0ae102.png">